Applications/Hermes/Readme.md

```diff
@@ -0,0 +1,105 @@
+# Hermes - Message Application
+
+Hermes is a simple messaging application developed for Faraday. Hermes was the God of messengers so it seemed fitting.
```

Comment:
Second sentence and beyond: "Hermes, the messenger of the gods in mythology, is a fitting name for our first application to transition into messaging between radios."